sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx

```diff
@@ -78,7 +78,15 @@ const NetworksSwitcher: FC<NetworksSwitcherProps> = () => {
 			</components.DropdownIndicator>
 		);
 	};
-
+	
+	const network = useRecoilValue(networkState);
+	
+	const getL2Network = () => {
+		if (network.id == 69) {
+			return `OΞ Kovan`;
+		} else return `Optimism`;
+	};
+	
```

Comment:
Suggestion:

By the time we get to this point in the function, we are mostly defining Components.  If we move this to the top of the function where the other values are assigned, and make use of the ternary operator, we can define it as as a one-liner. 

Note also: 
1. The string literals were changed to use single quotes in place of backticks to match the other string literals in the file.
2. As typeof `useRecoilValue(networkState).id` is `number`, I changed to strict equality to silence the lint warning.
3. Dropped the function wrapper and went for direct invokation as I didn't think the additional call was adding anything (I could be wrong)

```javascript
const NetworksSwitcher: FC<NetworksSwitcherProps> = () => {
	const { switchToL1, switchToL2 } = useNetworkSwitcher();
	const { t } = useTranslation();
	const isL2 = useRecoilValue(isL2State);
	const networkLabel = useRecoilValue(networkState).id === 69 ? 'OΞ Kovan' : 'Optimism';
	const theme = useTheme();
	const { blockExplorerInstance } = BlockExplorer.useContainer();

... then later ...

<L2Select
	formatOptionLabel={formatOptionLabel}
	controlHeight={28}
	options={OPTIMISM_OPTIONS}
	value={{ label: networkLabel, prefixIcon: 'Optimism' }}
...
```

I didn't include this, but you could also replace `69` with `NetworkId['Kovan-Ovm']` (will leave that call up to you/others) to remove the magic number/make the enum the source of truth.   However, it's fairly obvious in this case what 69 is referring to, and AFAIK that network ID won't change in the future.

Reply:
Thanks a lot for your extensive feedback. Agree with changing to a one-liner at the top of the function, totally makes sense now that you mentioned it.